src/core/modules/Constellation.cpp

```diff
@@ -91,7 +91,7 @@ bool Constellation::read(const QString& record, StarMgr *starMgr)
 		}
 		else
 		{
-			constellation[i]=starMgr->searchGaia(HP);
+			constellation[i]=starMgr->searchGaia(static_cast<int64_t>(HP));
```

Comment:
Here the type hasn't changed to `StarId`.

Reply:
fixed